### PR TITLE
Lecture 02-a

### PR DESCRIPTION
**Pull request by positivevaib**
Neural networks model the composition of linear and element-wise non-linear transformations, and so the calculation of gradients to be used in model training via the chain-rule is a natural choice. Backpropagation is the process of finding the gradients of the weights and biases of a neural network using chain rule, starting with the last layer, and moving backwards. It provides a disciplined way to train neural networks using gradient descent.

-- First part of Lecture 2 (4 February 2019) by Yair Schiff and Goutham Panneeru, edited by Vaibhav Gadodia

### CODE REVIEW

**Comment by Atcold** [comment]
Description and authors list are missing. Please, check previously accepted PR to get an example. Thanks.

**Comment by Atcold** [comment]
Thanks!
